navigator/runtime/src/main/java/com/freeletics/mad/navigator/internal/NavigationSetup.kt

```diff
@@ -51,8 +52,8 @@ private fun NavigationExecutor.navigate(
         is NavEvent.ActivityResultEvent<*> -> {
             val request = event.request
             val launcher = activityLaunchers[request] ?: throw IllegalStateException(
-                "No launcher registered for $request!\nMake sure you called the appropriate " +
-                    "NavEventNavigator.registerFor... method"
+                "No launcher registered for request with contract ${request.contract}!" +
+                    "\nMake sure you called the appropriate NavEventNavigator.registerFor... method"
```

Comment:
Makes the error message a bit more actionable since `request` itself is very non descriptive.